src/test/java/api/PrecedingSucceedingTitlesApiExamples.java

```diff
@@ -652,13 +635,23 @@ private JsonObject createSemanticWebUnconnectedTitle(String precedingSucceedingT
           .put("value", "0262012103")));
   }
 
-  private void assertPrecedingSucceedingTitles(JsonObject actualPrecedingTitle,
-    JsonObject expected, String precedingInstanceId, String succeedingInstanceId) {
+  private void assertPrecedingTitles(JsonObject actualPrecedingTitle,
+    JsonObject expected, String precedingInstanceId) {
 
     assertThat(actualPrecedingTitle.getString("title"), is(expected.getString("title")));
     assertThat(actualPrecedingTitle.getString("hrid"), is(expected.getString("hrid")));
     assertThat(actualPrecedingTitle.getJsonArray("identifiers"), is(expected.getJsonArray("identifiers")));
     assertThat(actualPrecedingTitle.getString("precedingInstanceId"), is(precedingInstanceId));
-    assertThat(actualPrecedingTitle.getString("succeedingInstanceId"), is(succeedingInstanceId));
+    assertNull(actualPrecedingTitle.getString("succeedingInstanceId"));
+  }
+
+  private void assertSucceedingTitles(JsonObject actualSucceedingTitle,
+    JsonObject expected, String succeedingInstanceId) {
+
+    assertThat(actualSucceedingTitle.getString("title"), is(expected.getString("title")));
+    assertThat(actualSucceedingTitle.getString("hrid"), is(expected.getString("hrid")));
+    assertThat(actualSucceedingTitle.getJsonArray("identifiers"), is(expected.getJsonArray("identifiers")));
+    assertThat(actualSucceedingTitle.getString("succeedingInstanceId"), is(succeedingInstanceId));
```

Comment:
I think it makes sense to also check that a connected succeeding title does not contain a `precedingInstanceId` as well.

Reply:
was added